engine/src/main/java/org/terasology/logic/chat/ChatSystem.java

```diff
@@ -94,25 +94,40 @@ public void onMessage(MessageEvent event, EntityRef entity) {
             shortDescription = "Sends a message to all other players")
     public String say(
             @Sender EntityRef sender,
-            @CommandParam(value = "message") String message
+            @CommandParam(value = "message") String[] message
     ) {
-        logger.debug("Received chat message from {} : '{}'", sender, message);
+        String messageToString = join(message, " ");
+
+        logger.debug("Received chat message from {} : '{}'", sender, messageToString);
 
         for (EntityRef client : entityManager.getEntitiesWith(ClientComponent.class)) {
-            client.send(new ChatMessageEvent(message, sender.getComponent(ClientComponent.class).clientInfo));
+            client.send(new ChatMessageEvent(messageToString, sender.getComponent(ClientComponent.class).clientInfo));
         }
 
         return "Message sent.";
     }
 
+    private String join(String[] strings, String cement) {
```

Comment:
@GabrielXia If you are interested in a nice one-line solution for this, using Java 8, have a look at this [Tweet](https://twitter.com/danieldietrich/status/429723665962655744). The code for `join` would look like this (I hope without errors):

```java
private String join(String[] words, String sep) {
  Arrays.stream(words).collect(joining(sep));
}
```

Feel free to open another PR if you want to change this ;-)